src/components/FiscalYear/FiscalYearDetails/FiscalYearDetailsContainer.js

```diff
@@ -72,8 +88,10 @@ const FiscalYearDetailsContainer = ({
   const removeFiscalYear = useCallback(
     () => {
       mutator.fiscalYear.DELETE(fiscalYear)
-        .then(() => history.goBack());
+        .then(showToast('ui-finance.fiscalYear.actions.remove.success'));
```

Comment:
perhaps, you want something like in invoice, within error notification https://github.com/folio-org/ui-invoice/blob/master/src/invoices/InvoicesList/Invoice/InvoiceDetailsLayer.js#L59

Reply:
👍 fixed